layers/meta-balena-jetson/recipes-core/images/balena-image.bbappend

```diff
@@ -22,6 +23,7 @@ IMAGE_ROOTFS_SIZE:jetson-orin-nx-seeed-j4012 = "733184"
 IMAGE_ROOTFS_SIZE:jetson-orin-nano-devkit-nvme = "733184"
 IMAGE_ROOTFS_SIZE:jetson-orin-nano-seeed-j3010 = "733184"
 IMAGE_ROOTFS_SIZE:jetson-agx-orin-devkit-64gb = "827392"
+IMAGE_ROOTFS_SIZE:forecr-dsb-ornx = "983040"
```

Comment:
@asuykov is the resulting flasher image lower than 4GB? If it's 4GB or higher, then zip compression won't work

Reply:
The resulting flasher image is 4.05GB. I tried zipping/unzipping it, works fine. At least with the Linux zip tool.

The size matches jetson-orin-nx-xavier-nx-devkit a couple of lines above that, so assuming the NX devkit configuration works this one should as well. This board takes NX along with Nano and I decided to keep the layout consistent for different modules on this board.

Reply:
Never mind, updated to 733184 (Seeed j4012/j3010).
Forecr NX configuration builds just fine, and the resulting image is well under 4GB now.